### ISSUE TITLE
AVT(JAWS) issue for focus in CC modal

### ISSUE THREAD

**Issue by shalaka-kshirsagar** [carbon-design-system/carbon#1075]
## Detailed description

> Describe in detail the issue you're having.

There is an AVT(JAWS tool) issue reported for modal.
         "When focus is on 'Close X' button on pressing shift+tab key, focus goes missing on the page.
Make sure focus is always visible on the page."
          CC modal is being used and the issue can be observed in http://www.carbondesignsystem.com/components/modal/code

> Is this issue related to a specific component?

CC modal

> What did you expect to happen? What happened instead? What would you like to see changed?

When focus is on the first focusable element in the dialog, moves focus to the last focusable element in the dialog on pressing shift+tab. As mentioned in https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

> What browser are you working in?

Firefox,chrome

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

Cloud Automation Manager

To reproduce and check the issue, we do not need JAWS and should simply work with tab and shift+tab

<img width="1118" alt="screen shot 2018-09-10 at 1 14 18 pm" src="https://user-images.githubusercontent.com/40288101/45315058-f9566500-b550-11e8-9064-58ff2883943b.png">

**Comment by shalaka-kshirsagar** [comment]
We would be obliged if this issue would be addressed as early as possible due to a strict deadline assigned to us for the delivery of AVT changes. Thank you.

**Comment by asudoh** [comment]
Hi :wave: while implementing focus-wrap support back-tab introduces a problem of defining last focusable element in a modal, you can address the problem by overriding [`_handleFocusin()` method](https://github.com/IBM/carbon-components/blob/v9.26.0/src/components/modal/modal.js#L141-L154) of our `Modal` JavaScript class and do what you desire. Thanks!

**Comment by dakahn** [comment]
closing this as resolved 🤠
